patch-diff-report-tool/src/main/java/com/github/checkstyle/data/DiffReport.java

```diff
@@ -76,6 +82,7 @@ public Statistics getStatistics() {
     public void addRecords(List<CheckstyleRecord> newRecords,
             String filename) {
         if (!newRecords.isEmpty()) {
+            Collections.sort(newRecords);
```

Comment:
As mentioned in previous PR, why not sort the records as they are added? Is that faster?

Reply:
Nevermind. https://stackoverflow.com/questions/168891/is-it-faster-to-sort-a-list-after-inserting-items-or-adding-them-to-a-sorted-lis